examples/md-flexible/src/domainDecomposition/RegularGridDecomposition.cpp

```diff
@@ -570,6 +571,8 @@ RegularGridDecomposition::categorizeParticlesIntoLeftAndRightNeighbor(const std:
         const auto justInsideOfBox = std::nextafter(_globalBoxMax[direction], _globalBoxMin[direction]);
         position[direction] = std::min(justInsideOfBox, periodicPosition);
         leftNeighborParticles.back().setR(position);
+        leftNeighborParticles.back().setRAtRebuild();  // TODO: this isn't correct, best thing would be to shift so that
+                                                       // displacement before and after remains same
```

Comment:
```suggestion
        // TODO: this isn't correct, best thing would be to shift so that displacement before and after remains same
        leftNeighborParticles.back().setRAtRebuild();  
```

- [ ] Reformat so that comments are above the line they refer to
- [ ] Reformulate what exactly is not correct
- [ ] Is this something we should fix? Then create an issue. If not then this shouldn't probably even be marked as todo but just as an simplification

Reply:
resolved with in-person discussion